versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.0, < 4.0.0"
+      version = ">= 4.9.0"
     }
     local = {
       source  = "hashicorp/local"
```

Comment:
To get tests to detect the correct version of terraform, copy `versions.tf` to `examples/complete`